src/Policies/Policy.php

```diff
@@ -55,12 +53,14 @@ public function applyTo(Response $response): void
     public function __toString()
     {
         return collect($this->directives)
-            ->map(function (array $values, string $directive) {
-                $valueString = implode(' ', $values);
+            ->map(function (array $rules, string $directive) {
+                $formattedRules = implode(' ', $rules);
+
+                if (count($rules) === 1) {
```

Comment:
refactored the tenary to use a guard statement. Think that this makes it more clear what the difference is between the two